### ISSUE TITLE
Add `seff` to Toil Slurm?

### ISSUE THREAD

**Issue by adamnovak** [DataBiosphere/toil#4501]
The `seff` command can get statistics about how much CPU and memory a jib actually used.

It might be useful for Toil to expose this information in its logs somehow when running on Slurm, either to diagnose OOM or to diagnose excessively large job resource allocations.

**Comment by mr-c** [comment]
Yes, this would be good to have; and [report as part of RO-Crate WorkflowRun provenance](https://github.com/ResearchObject/workflow-run-crate/issues/10)

**Comment by adamnovak** [comment]
This might also be useful for Dockstore as part of a workflow analytics report-back feature/a bigger system.
